vyper/codegen/expr.py

```diff
@@ -661,25 +665,27 @@ def parse_UnaryOp(self):
             min_int_val, _ = operand.typ.int_bounds
             return IRnode.from_list(["sub", 0, clamp("sgt", operand, min_int_val)], typ=operand.typ)
 
+    def parse_Await(self):
```

Comment:
```suggestion
    def parse_Extcall(self):
```
Should this be split into `parse_Extcall` and `parse_Staticcall`?

Reply:
ditto, dead function - ExtCall and StaticCall are a few lines down.